frontend/src/pages/SignupPage/SignupPage.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import axios, { AxiosError,HttpStatusCode } from 'axios';
+import { ToastContainer, toast } from 'react-toastify';
+import { AUTH_REGISTER_URL } from '../../constants/constants';
+import SignupFormData from '../../interfaces/SignupFormData';
+import isEmailValidCheck from '../../utils/isEmailValidCheck';
+
+import 'react-toastify/dist/ReactToastify.css';
+import './SignupPage.css'
+
+const SignupPage = () => {
+  const [signupFormData, setSignupFormData] = useState<SignupFormData>({
+    username: '',
```

Comment:
```suggestion
    userName: '',
```
To fix this, you can either update the frontend code to use `userName` instead of `username`